### DISCUSSION TITLE
Configure LSPs to look for local command in $PATH?

### DISCUSSION THREAD

**Discussion by benfrain** [zed-industries/zed#21747] [Feature Requests]
I have the language servers I need installed locally and accessible in my $PATH. Can I configure Zed to use these LSPs? Should something like this work for CSS?

```
"lsp": {
  "cssls": {
    "initialization_options": {
      "check": {
        "command": "~/.local/share/nvim/spec/node_modules/.bin/vscode-css-language-server --stdio"
      }
    }
  }
},
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
